solar_crypto/identity/private_key.py

```diff
@@ -46,7 +48,7 @@ def from_passphrase(cls, passphrase):
         return cls(private_key)
 
     @classmethod
-    def from_hex(cls, private_key):
+    def from_hex(cls, private_key) -> PrivateKey:
```

Comment:
The return type is great, but the parameter should probably be `str` too, shouldn't it?

Reply:
No, because it returns the `PrivateKey` object